hermes/file_gen_test.go

```diff
@@ -47,13 +28,32 @@ func TestChecksum(t *testing.T) {
 	}
 }
 
-// Test function for the generateFileName function in file_gen.go
+type FileNameTestUnit struct {
+	id       int
+	checksum string
+	want     string
+}
+
+var test_unit_table [5]FileNameTestUnit
+
 func TestFileName(t *testing.T) {
```

Comment:
Nit: it's normal to define this struct in line, which makes this somewhat more readable.